lib/facter/resolvers/networking_linux.rb

```diff
@@ -290,6 +291,13 @@ def retrieve_primary_interface
 
           @fact_list[:primary_interface] = primary_interface
         end
+
+        def retrieve_primary6_interface
+          primary_helper = Facter::Util::Resolvers::Networking::PrimaryInterface
+          primary6_interface = primary_helper.read_from_ip6_route
+
+          @fact_list[:primary6_interface] = primary6_interface
```

Comment:
I would suggest, if that's ok with you, adding some fallbacks in case the `read_from_ip6_route ` doesn't retrieve anything. Like we do inside the `retrieve_primary_interface` method. Maybe at least setting the primary6 as the first not ignored ipv6 address?